charts/seed-operatingsystemconfig/original/templates/var-lib-mount/var-lib.mount

```diff
@@ -0,0 +1,14 @@
+{{- define "var-lib-mount" -}}
```

Comment:
This refers to @majst01's comment (https://github.com/gardener/gardener/pull/1893#issuecomment-582872196): Shouldn't this mount by only performed conditionally due to the standard behavior we already have today?

Reply:
I made the unit conditional, same as the addition to _`cloud-config-script.sh`.